docs/docusaurus.config.js

```diff
@@ -1,78 +1,82 @@
 module.exports = {
   title: "Aurora",
-  tagline: "Aurora Design Guidelines by Grape",
-  url: "https://aurora.grape.io",
+  tagline: "Grape's design system",
+  url: "https://aurora.ubergrape.com",
   baseUrl: "/",
   onBrokenLinks: "throw",
   favicon: "img/favicon.ico",
-  organizationName: "ubergrape",
+  organizationName: "UberGrape GmbH",
   projectName: "aurora",
   themes: ["@docusaurus/theme-live-codeblock"],
   themeConfig: {
+    colorMode: {
+      defaultMode: "light",
+      disableSwitch: true,
+    },
     navbar: {
       title: "Aurora",
       logo: {
-        alt: "Aurora Logo",
+        alt: "Grape logo",
         src: "img/logo.svg",
       },
       items: [
         {
           to: "docs/",
           activeBasePath: "docs",
-          label: "Docs",
-          position: "left",
+          label: "Design",
+          position: "right",
         },
         {
-          href: "https://github.com/ubergrape/grape-ds",
-          label: "GitHub",
+          href: "https://aurora.ubergrape.com/storybook/",
+          label: "Development 🡕",
           position: "right",
         },
       ],
     },
     footer: {
-      style: "dark",
+      style: "light",
       links: [
         {
           title: "Docs",
           items: [
             {
-              label: "Style Guide",
+              label: "Design",
               to: "docs/",
             },
             {
-              label: "Second Doc",
-              to: "docs/doc2/",
+              label: "Development 🡕",
```

Comment:
what character is that? I just see a questionmark in my browser and my text editor.

![Screenshot 2020-12-15 at 16 02 08](https://user-images.githubusercontent.com/385177/102232391-5808a700-3eef-11eb-8581-2c50e73ef262.png)

Reply:
It's an arrow, like 🡕 (up right), indicating external link.
Looks fine on Chrome and Firefox for me.
![pseudoshot-0007](https://user-images.githubusercontent.com/1522647/102234809-f5fd7100-3ef1-11eb-8a91-85a3214816a6.png)

Reply:
ok this is in unicode 7 and macos should support at least unicode 8 since macos 10.11 or so (i'm on 10.14.6)
https://www.compart.com/de/unicode/U+1F855
Oleh also sees a square on his mac

on this page it renders because of the font-family rule
https://unicode-table.com/de/1F855/
`<span class="vchar" id="char-node" style="font-family:u1f800;">🡕</span>`

Reply:
@sk7 Removed the arrow from the text now as it's not critical to have it.